setup.md

```diff
@@ -91,11 +91,13 @@ GitHub is a free, online host for Git repositories that you will use during the
 You can create an account at [https://github.com/](https://github.com/) if you don't already have one.
 
 ## Anaconda Python Distribution
-We will be using the Anaconda Python Distribution for the workshop. The material is not tested with other Python distributions and is specific to Anaconda.
+The material has been developed using the [Anaconda Python distribution](https://www.anaconda.com/) and is using the `conda` package manager 
+that comes with Anaconda. The material was not tested with other Python distributions and package managers but could 
```

Comment:
How about, "The material has not been extensively tested with other Python distributions and package managers, but most sections are expected to work with some modification. For example, package installation and virtual environments would need to be managed differently."

Reply:
I agree we should push people to use conda, but if they're already familiar with `pip`, `venv`, etc. and know how to translate the notes for them, then they shouldn't be too disruptive.

The usage of conda in the CI section is probably the most difficult to replace, but it should work if they used system Python locally and conda for CI.

Reply:
Fixed via e67a345.

Reply:
Merging this branch.